swiftpkg/internal/swiftpkg_build_files.bzl

```diff
@@ -1055,7 +1080,9 @@ def _new_for_license(pkg_info, license):
 
 # MARK: - Constants and API Definition
 
-swift_location = "@build_bazel_rules_swift//swift:swift.bzl"
+swift_repository = "@build_bazel_rules_swift"
+swift_location = "{rules_swift}//swift:swift.bzl".format(rules_swift = swift_repository)
+no_module_aspect_hint_label = "{rules_swift}//swift:no_module".format(rules_swift = swift_repository)
```

Comment:
The `no_module_aspect_hint_label` does not appear to be used. Can we revert these three lines?

Reply:
Oops leftover from testing Brentley's suggestion, removed these changes.